samplesheets/urls.py

```diff
@@ -193,6 +193,11 @@
         view=samplesheets.views_api.RemoteSheetGetAPIView.as_view(),
         name='api_remote_get',
     ),
+    url(
+        regex=r'^api/irods/files/(?P<project>[0-9a-f-]+)',
```

Comment:
To make this more consistent with API view URL paths, we should probably use `api/file/list/` etc.